modules/distr-tracing-otel-config-collector.adoc

```diff
@@ -365,3 +365,56 @@ The Logging exporter prints data to the standard output.
           exporters: [logging]
 ----
 <1> Verbosity of the logging export: `detailed` or `normal` or `basic`. When set to `detailed`, pipeline data is verbosely logged. Defaults to `normal`.
+
+[id="extensions_{context}"]
+=== Extensions
+
+[id="basicauth-extension_{context}"]
+==== BASICAUTH extension
+
+The BasicAuth extension can be used as an authenticator on receivers and exporters.
+The configuration must specify only one instance of the BasicAuth extension: for client authentication or server authentication.
+
+* Support level: link:https://access.redhat.com/support/offerings/techpreview[Technology Preview]
+* Supported signals: traces, metrics, logs
+
+.OpenTelemetry Collector custom resource with a client and server basicauth extension
+[source,yaml]
+----
+  config: |
+    extensions:
+      basicauth/server:
+        htpasswd:
+          file: .htpasswd <1>
+        inline: |
+          ${env:BASIC_AUTH_USERNAME}:${env:BASIC_AUTH_PASSWORD} <2>
+
+      basicauth/client:
+        client_auth:
+          username: username <3>
+          password: password <4>
+
+    receivers:
+      otlp:
+        protocols:
+          http:
+            auth:
+              authenticator: basicauth/server <5>
+    exporters:
+      otlp:
+        auth:
+          authenticator: basicauth/client <6>
+
+service:
+  extensions: [basicauth/server, basicauth/client]
+  pipelines:
+    traces:
+      receivers: [otlp]
+      exporters: [otlp]
+----
+<1> The basicauth extension getting configured as a server authenticator reading credentials from a `.htpasswd` file.
+<2> The basicauth extension getting configured as a server authenticator reading the credentials from an inline string. This string consists of the read-in environment variables `BASIC_AUTH_USERNAME` and `BASIC_AUTH_PASSWORD`.
```

Comment:
```suggestion
<2> Here the BasicAuth extension is configured as a server authenticator that reads the credentials from an inline string. This string consists of the  `BASIC_AUTH_USERNAME` and `BASIC_AUTH_PASSWORD` read-in environment variables.
```

Reply:
```suggestion
<2> Here the BasicAuth extension is configured as a server authenticator that reads the credentials from an inline string. This string consists of the `BASIC_AUTH_USERNAME` and `BASIC_AUTH_PASSWORD` read-in environment variables.
```

Reply:
this inline field you can see in the configuration consists out of a username and password. e.g. `tom:123` username tom, password 123. In the given example we used `{$env:BASIC_AUTH_USERNAME}` as username. This will read the username from an environment variable named `BASIC_AUTH_USERNAME`.
Maybe the word `defined` is a bit better?

Reply:
Thank you for clarifying this. I've updated my suggestion above.

Reply:
```suggestion
<2> Here the BasicAuth extension is configured as a server authenticator to read the credentials from an inline string consisting of environment variables.
```

Reply:
Does this work too? Just to make clear that the string is build out of the env variables.

Reply:
I updated my suggestion in https://github.com/max-cx/openshift-docs/pull/13#discussion_r1334085609 to state "inline string that consists of". Is this acceptable to you?

Reply:
perfect :)